### PR TITLE
Create allmight.yml

### PR DESCRIPTION
**Pull request by MH03**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/chars/allmight.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class: allmight
+costumes:
+- chatcolor: '&6'
+  name: §6All§4Mi§1gh§6t
+  sig: R3HIC3T/kTdslaIAJVrREV8sU4K2Jg3rL6HwUxQhiSxK1BV67rJMmx1sDB4ZexCT9IwLVh8zeNN8CVKaYdPZUX/ahRb2QpRjLzO0bcaZZRZ5ySS80zbT5xP7FHNSLKtsiyLu6t+zqm7n5s6m7jpen28/jDkZttjBIwv+J4NbKsgf2XI8oFiJcPu1YDgaF8B0Ou35Mrv0M0y1n3Yl75Jm0KRxaVkTxN1N3IVQWOupBOzdK9frUBmr2AQAnmnXgcnM1go892PDcneUaUG6bwrAvh4ednG6f1FfbeNozeLkmxytrLtSaGoWBwDulTuWWMm1N44eimdnOabDGqnr64IuRykNfSQYCztr5kLxTfOle1LTxwa3F7Guz427zxnsMIvdlyBsryWDYyiP9b0OqGuGgnr0YGaKidbhX7CxBzqRmYfE9IVkwQDvWNdivMahw0rwPK5uAaOIobZHnk3FOOZgCWa+jMdQkFh3fyzN28FLkYxmMAiNt4/e8O0l64dUqzwsTULPiVt5briThal1cf1ZEIpsm017cldaDA8giadak00kuhY/TX6qPENotsIJayK3m20MFg3ucoxRXYdFhdCTOssT2G9kyO4Gna32zXQU6jT7PLqbcpy6aJRoMmnSyarPu62D6T8uel/MLZUeFDTPlLbfn0G+o9wPXalJYd9NRCM=
+  val: eyJ0aW1lc3RhbXAiOjE1MjIwMTI2MTQxMjUsInByb2ZpbGVJZCI6IjQ3NDcwN2Y3ZjhiNDQzZWE4OTc4OWNhZTE3OTE3ZTgyIiwicHJvZmlsZU5hbWUiOiJUaGF0Q3JhZnRMaWZlTUNVIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83Y2FhNDY0YjY3YjhiMzJiMjg1ZTNjNDVlYjdlN2IxMDE5YTA1NmE1NzI5ODgxZDRiYjJiNzFlZThkMTc1OCJ9fX0=
+effects:
+- /effect <plr> 5 9999 6 true
+- /effect <plr> 11 9999 2 true
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/allmight.yml]: proof required for these effects

File: CommandHelper/LocalPackages/commands/chars/allmight.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+haste: 9
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/allmight.yml]: proof required for haste 9

File: CommandHelper/LocalPackages/commands/chars/allmight.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+items: [
+  ]
+jump: 9
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/allmight.yml]: proof required for jump 9

File: CommandHelper/LocalPackages/commands/chars/allmight.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  superherocity:
+  - ch.alias.dropplayer
+  - ch.alias.carryplayer
+powers:
+- /effects
+- /items
+- /jump <0-9>
+- /run <0-9>
+- /haste <0-9>
+run: 9
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/allmight.yml]: proof required for run 9

File: CommandHelper/LocalPackages/commands/chars/allmight.yml (hunk 1 above)
Comment (MH03) [reply CommandHelper/LocalPackages/commands/chars/allmight.yml]: https://www.youtube.com/watch?v=6m_oxvyIde0 
Resistance 0:13
Strength  0:43

File: CommandHelper/LocalPackages/commands/chars/allmight.yml (hunk 4 above)
Comment (MH03) [reply CommandHelper/LocalPackages/commands/chars/allmight.yml]: https://www.youtube.com/watch?v=6m_oxvyIde0
0:18

**Pull request merged**